src/com/lftechnology/java/training/srijan/basics/coreapp/ui/EmployeeView.java

```diff
@@ -0,0 +1,397 @@
+			employees = employeeControl.viewAllUser(em);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		}
+	}
+
+	/**
+	 * <p>edit and terminate user</p>
+	 * 
+	 * @return
+	 * @author srijan
+	 */
+	public int editAndTerminateEmployee(Scanner input) {
+
+		int employeeIdToEdit = 0;
+		try {
+			input = new Scanner(System.in);
+			employeeIdToEdit = input.nextInt();
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception", ex);
+		}
+		return employeeIdToEdit;
+	}
+
+	/**
+	 * <p>Change profile of an user</p>
+	 * 
+	 * @param emp
+	 * @return
+	 * @author srijan
+	 */
+	public Employee changeProfile(Employee emp,Scanner input) {
+
+		try {
+			input = new Scanner(System.in);
+			LOGGER.log(Level.INFO, "Enter Username:");
+			this.username = input.nextLine();
+			if (!"".equals(username))
+				emp.setUserName(username);
+			LOGGER.log(Level.INFO, "Enter Fullname:");
+			this.fullname = input.nextLine();
+			if (!"".equals(fullname))
+				emp.setFullName(fullname);
+			LOGGER.log(Level.INFO, "Enter Department:");
+			this.department = input.nextLine();
+			if (!"".equals(department))
+				emp.setDepartment(department);
+			LOGGER.log(Level.INFO, "Enter Address:");
+			this.address = input.nextLine();
+			if (!"".equals(address))
+				emp.setAddress(address);
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception{0}", ex);
+		}
+		return emp;
+	}
+
+	/**
+	 * <p>Takes input from user the search choice and value of choice</p>
+	 * 
+	 * @return
+	 */
+	public SearchEmployee choiceOfSearch(Scanner input) {
+
+		SearchEmployee emp = new SearchEmployee();
+		int choice = 0;
+		try {
+			input = new Scanner(System.in);
+			choice = input.nextInt();
+			if (choice == 1)
+				emp.setSearchChoice("department");
+			else if (choice == 2)
+				emp.setSearchChoice("fullname");
+			else if (choice == 3)
+				emp.setSearchChoice("address");
+			else
+				emp.setSearchChoice("default");
+			
+			emp = valueOfSearchChoice(emp,input);
+			
+		}catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception{0}", ex);
+		}
+		return emp;
+	}
+	
+	public SearchEmployee valueOfSearchChoice(SearchEmployee emp,Scanner input){
+		String choiceValue = "";
+		try{
+			input = new Scanner(System.in);
+		if (!emp.getSearchChoice().equals("default")) {
+			LOGGER.log(Level.INFO, "Write {0}?", emp.getSearchChoice());
+			choiceValue = input.nextLine();
+			if (!"".equals(choiceValue)) {
+				emp.setChoiceValue(choiceValue);
+			}
+		}	
+		}catch(Exception ex){
+			LOGGER.log(Level.INFO, "Exception{0}", ex);
+		}
+		return emp;
+	}
+
+	/**
+	 * <p>Displays various options for ADMIN user</p> <p>Inputs choice from
+	 * ADMIN user</p>
+	 * 
+	 * @param controller
+	 * @author srijan
+	 */
+	public void adminAccess(Scanner input) {
+
+		Boolean looping = true;
+		do {
+			int userInput = 0;
+			LOGGER.log(
+				Level.INFO,
+				"\n1.Add User\n2.Search User \n3.Terminate User\n4.View Employee\n5.Exit");
+			try {
+				input = new Scanner(System.in);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "input". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)